ores/wsgi/util.py

```diff
@@ -61,3 +63,18 @@ def format_output(context, scores, model_info, warning=None, notice=None):
         for rev_id in scores[model]:
             output['scores'][context][model]['scores'][rev_id] = scores[model][rev_id]
     return jsonify(output)
+
+
+# Parse values for features / datasources of interest.
```

Comment:
Good Q.  Don't worry about pep 8 issues on the rest of the file structure (unless you want to).  We can get those in a follow up PR.